pkgs/development/compilers/rust/rustc.nix

```diff
@@ -97,11 +97,11 @@ stdenv.mkDerivation {
     rm -vr src/test/run-make/linker-output-non-utf8 || true
 
     # Remove test targeted at LLVM 3.9 - https://github.com/rust-lang/rust/issues/36835
-    rm -vr src/test/run-pass/issue-36023.rs || true
+    rm -fvr src/test/run-pass/issue-36023.rs
```

Comment:
Yup, I know. Maybe I get around writing a patch to remove these files (or better: add the `# ignore` line). Do you know which file needs to be removed for which compiler version?